README.md

```diff
@@ -50,15 +50,15 @@ may also provide configurations with services split among multiple servers.
 |                   |
 |   Raiden clients  |
 |                   |
-+---+-----------+---+
-    |matrix://  |pfs://
-====|===========|====
-    |           |
-+---v-----------v---+                       Federation to
++---------+---------+
+          |https://
+==========|==========
+          |
++---------v---------+                       Federation to
 |                 +-+-------------------->  other Raiden
 |      Traefik    | |                       Matrix servers
-|                 | |
-+---------+-------+-+---------+
+|                 |-+---------+
```

Comment:
The Monitoring Service (or to be precise, its companion process the Request Collector) also reads Matrix messages. Unsurprisingly, it is interested in the MonitoringRequest messages.

Reply:
Ah good point, will update.